src/components/AppContent/AppContent.vue

```diff
@@ -40,22 +107,71 @@ import { emit } from '@nextcloud/event-bus'
 export default {
 	name: 'AppContent',
 
+	components: {
+		Splitpanes,
+		Pane,
+	},
+
 	props: {
 		// Allows to disable the control by swipe of the app navigation open state
 		allowSwipeNavigation: {
 			type: Boolean,
 			default: true,
 		},
+		// Allows you to set the default width of the resizable list in %
+		// Must be between listMinWidth and listMaxWidth
```

Comment:
```suggestion
		/**
		 * Allows you to set the default width of the resizable list in %
		 * Must be between listMinWidth and listMaxWidth
		 */
```
This way it is parsed correctly by netlify and appears in the docs.